### ISSUE TITLE
Session Storage empty after short request intervals

### ISSUE THREAD

**Issue by gbaumgart** [nette/nette#1543]
Hi,
when the time between my requests is too small (less than a sec), Nette seems to loose my session storage (in a section). However, when re-request again after 3 secs or so, its back again.

What could this be?
I am using Nette 2.1

**Comment by gbaumgart** [comment]
Ok, I updated to 2.3.8 but the problem remains. 
I creates the Session and the Section object as advised. Could this be a limitation of PHP or Nette? I really don't know what to do know, I tried all sort of things like $section->setExpiration(10000) and also play with the options for the sessions. 

Can anyone confirm this issue? We're really stuck on this and we have to keep moving, despite loosinga lot of work around nette now.

well, any thoughts are welcome
